.gitignore

```diff
@@ -12,4 +12,7 @@ vendor/
 /ignore
 
 # Ignore "app.yaml" because it's stored as a secret in GCPSM
-app.yaml
\ No newline at end of file
+app.yaml
+
+# DB stuff
+/local-db/*
```

Comment:
We'll still need to track the /local-db folder since it has the Dockerfile and is referenced in the README with the DB SQL copy/download command.

Reply:
Maybe explicitly ignore `/local-db/skyviewer.sql`?

Reply:
good point